pkg/devices/fan.go

```diff
@@ -85,6 +85,8 @@ func fanWrite(device *sdk.Device, data *sdk.WriteData) error {
 	switch data.Action {
 	case "speed":
 		emitter.Set(v)
+	case "speed_percent":
+		emitter.Set(v)
```

Comment:
these changes  work, but could be written more concisely since they are running the same code within the case blocks

```go
case "speed", "speed_percent":
    emitter.Set(v)
```